File: django_backend/django_backend/models/book.py
from django.db import models
from django_backend.models import (
    BookDescription
)


class Book(models.Model):
    class BookState(models.TextChoices):
        NEW = "0", "Новая"
        GOOD = "1", "Хорошая"
        WORN = "2", "Поношенная"
        UNDER_REPAIR = "3", "В ремонте"
        WRITTEN_OFF = "4", "Списана"
        LOST = "5", "Утеряна"

    id = models.AutoField(primary_key=True)
    inventory_number = models.CharField(unique=True, blank=False, null=False, db_index=True)
    description = models.ForeignKey(
        BookDescription,
        null=False,
        on_delete=models.PROTECT,
        related_name="books"
    )
    state = models.CharField(
        max_length=1,
        choices=BookState.choices,
        default=BookState.GOOD, 
        db_index=True
    )

File: react_frontend/src/dicts/book.ts
import { BookState } from "redux/types/book";

export const BOOK_STATE_NAMES: {[key in BookState]: string} = {
    "0": "Новая",
    "1": "Хорошая",
    "2": "Поношеная",
    "3": "В ремонте",
    "4": "Списана",
    "5": "Утеряна",
}